client/src/AlertsPage.jsx

```diff
@@ -5,17 +5,25 @@ import Cookies from 'js-cookie'
 
 const AlertsPage = ({ userId, hobbyName, hobbyId, fetchNotifications, notifications, setNotifications }) => {
     const intHobbyId = parseInt(hobbyId)
+    const [filter, setFilter] = useState("all")
```

Comment:
[nit] It'd be better if you can make this a Enum (feel free to address this later, no need to be in this PR)